src/Components/CreateAntFields/CreateAntUploader.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Form, Upload, Icon, Button, message } from "antd";
+import { FieldContainer } from './styled';
+
+const FormItem = Form.Item;
+
+const CreateAntUploader = () => ({
```

Comment:
Put this code to `components/formControls/creators/createUploader.ts`

Reply:
Done